src/monattack.cpp

```diff
@@ -1096,6 +1141,9 @@ void mattack::fungus_growth(monster *z, int index)
 
 void mattack::fungus_sprout(monster *z, int index)
 {
+    if( z->friendly ) {
+        return; // TODO: handle friendly monsters
```

Comment:
A Mycus in the area is no reason to stop maintaining the fungal wall.  Should proceed as normal.